debian/watch

```diff
@@ -0,0 +1,8 @@
+# See uscan(1) for format
```

Comment:
I was talking about the comment line being superfluous but if it's good to remember how to use it, then it should be fine.